oac_summary.m4

```diff
@@ -78,18 +78,16 @@ AC_DEFUN([OAC_SUMMARY_ADD],[
 dnl If $1 is "stderr", then send the output to stderr.  Otherwise,
```

Comment:
This file's changes should be in the previous commit.

Reply:
🤬  Doh!